.github/workflows/docker-image.yml

```diff
@@ -126,19 +125,30 @@ jobs:
           java-version: '8'
           distribution: 'temurin'
 
+      - name: Check groups of runner
+        run: |
+          groups runner
+      - name: Create user and set permissions
+        run: |
+          sudo useradd ranger
+          sudo usermod -aG docker ranger
+          sudo usermod -aG adm ranger
```

Comment:
where does the 'adm' group comes from?

Reply:
Removed, these permission setting is not needed anymore.